README.md

```diff
@@ -1,135 +1,410 @@
+     networks: {
+       kadena_devnet0: {
+         url: 'http://localhost:8545',
+         chainId: 1789,
+         accounts: devnetAccounts.accounts.map(account => account.privateKey),
+         chainwebChainId: 0,
+       },
+       kadena_devnet1: {
+         // url: 'http://localhost:8546',
+         url: 'http://localhost:8555',   
+         chainId: 1790,
+         accounts: devnetAccounts.accounts.map(account => account.privateKey),
+         chainwebChainId: 1,
+       },
+     },
+     ```
+
+1. (Optional) Copy and paste the code to configure etherscan settings from the `solidity/hardhat.config.js` file into the `hardhat.config.js` file for your project.
+   
+   For example:
+
+   ```javascript
+     // defaultNetwork: "kadena_devnet0",
+     etherscan: {
+       apiKey: {
+         'kadena_devnet0': 'empty',
+         'kadena_devnet1': 'empty',
+       }, 
+       customChains: [
+         {
+           network: "kadena_devnet0",
+           chainId: 1789,
+           urls: {
+             apiURL: "http://localhost:8000/api",
+             browserURL: "http://localhost:8000"
+           }
+         },
+         {   
+           network: "kadena_devnet1",
+           chainId: 1790,
+           urls: {
+             apiURL: "http://localhost:8001/api",
+             browserURL: "http://localhost:8001"
+           }
+         },
+       ]
+     },
+     ```
+1. Save your changes and close the `hardhat.config.js` file.
+
+### Compiling and testing integration
+
+After configuring your Hardhat project to use the Chainweb EVM development network, you can compile, test, and deploy the project using standard `hardhat` commands.
+For example, compile the project by running the following command:
 
 ```sh
-./network devnet pull
+npx hardhat compile
 ```
 
-Start the network:
+Run unit tests using the standard `hardhat test` command:
 
 ```sh
-./network devnet start
+cd solidity 
+npx hardhat test
 ```
 
-Check that blocks are being produced:
+Similarly, if you have a `deploy.js` deployment script in a `scripts` directory, you can deploy the project using the standard `hardhat` command to run scripts:
 
 ```sh
-./network devnet status
+npx hardhat run scripts/deploy.js
 ```
 
-Call the previous command repeatedly and check that the cut height value is
-increasing.
+### Changing the default network
 
-Explore available commands for managing and monitoring the network:
+If you prefer to keep the `hardhat` network as the default network for your Hardhat project, you can configure `kadena_devnet` as a separate network.
 
-```sh
-./network help
-```
+To change the default network:
 
-When done shut down the network with:
+1. Open the `hardhat.config.js` file for your Hardhat project in your code editor. 
 
-```sh
-./network devnet stop
-```
+2. Remove or comment out the following line from the network settings:
 
-If, for some reason, the network gets stuck the chainweb-node can be restarted
-via:
+   ```sh
+   defaultNetwork: "kadena_devnet",
```

Comment:
This has to refer to a network that is configured, so it should be `kadena_devnet0`